Forward the detected system Boost library suffix to Orcus, so its older
Boost macros can find architecture-tagged MacPorts libraries.

--- a/external/liborcus/ExternalProject_liborcus.mk
+++ b/external/liborcus/ExternalProject_liborcus.mk
@@ -85,6 +85,7 @@
 		$(if $(liborcus_CXXFLAGS),CXXFLAGS='$(liborcus_CXXFLAGS)') \
 		$(if $(liborcus_CPPFLAGS),CPPFLAGS='$(liborcus_CPPFLAGS) $(gb_EMSCRIPTEN_CPPFLAGS)') \
 		$(if $(liborcus_LDFLAGS),LDFLAGS='$(liborcus_LDFLAGS)') \
+		$(if $(SYSTEM_BOOST),boost_last_suffix='x$(patsubst -lboost_iostreams%,%,$(BOOST_IOSTREAMS_LIB))') \
 		MDDS_CFLAGS='$(MDDS_CFLAGS)' \
 		MDDS_LIBS=' ' \
 		MAKE=$(MAKE) $(gb_RUN_CONFIGURE) ./configure \
